guides/v2.3/graphql/develop/resolvers.md

```diff
@@ -5,34 +5,7 @@ title: Resolvers
 
 A resolver performs GraphQL request processing. In general, it is responsible for constructing a query, fetching data and performing any calculations, then transforming the fetched and calculated data into a GraphQL array format. Finally, it returns the results wrapped by a callable function.
 
-A resolver requires the following arguments:
-
--  $field
--  $context
--  $info
--  $value
--  $args
-
-```php
-    /**
-     * Fetches the data from persistence models and format it according to the GraphQL schema.
-     *
-     * @param \Magento\Framework\GraphQl\Config\Element\Field $field
-     * @param ContextInterface $context
-     * @param ResolveInfo $info
-     * @param array|null $value
-     * @param array|null $args
-     * @throws \Exception
-     * @return mixed|Value
-     */
-    public function resolve(
-        Field $field,
-        $context,
-        ResolveInfo $info,
-        array $value = null,
-        array $args = null
-    );
-```
+A GraphQL request is represented by the following arguments to processed by a resolver:
```

Comment:
Is this sentence introducing an example or stating a fact about all GraphQL requests? If it's introducing an example, then the beginning of the sentence should be reworded to make that more clear.

> The example in this topic shows how a resolver processes a GraphQL request that contains the following arguments: 

Either way, the end of the sentence needs some work:

> A GraphQL request is represented by the following arguments, **which will be** processed by a resolver:

Reply:
fixed